moveit_commander/test/python_moveit_commander_ns.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+import unittest
+import numpy as np
+import rospy
+import rostest
+import os
+
+from moveit_commander import RobotCommander
+
+
+class PythonMoveitCommanderNsTest(unittest.TestCase):
+    PLANNING_GROUP = "manipulator"
+    PLANNING_NS = "test_ns/"
+
+    @classmethod
+    def setUpClass(self):
+        self.commander = RobotCommander("%srobot_description"%self.PLANNING_NS, self.PLANNING_NS)
+        self.group = self.commander.get_group(self.PLANNING_GROUP)
+
+    @classmethod
+    def tearDown(self):
+        pass
+
+    def check_target_setting(self, expect, *args):
+        if len(args) == 0:
+            args = [expect]
+        self.group.set_joint_value_target(*args)
+        res = self.group.get_joint_value_target()
+        self.assertTrue(np.all(np.asarray(res) == np.asarray(expect)),
+                        "Setting failed for %s, values: %s" % (type(args[0]), res))
+
+    def test_target_setting(self):
```

Comment:
rename something like `test__group__set_joint_value_target__methods` to make it easier to understand what this test does